components/skin/ui/src/main/resources/PhenoTips/Skin.xml

```diff
@@ -782,24 +782,18 @@ h1:target, h2:target, h3:target {
   display: none;
 }
 .language-block {
-  background-color: transparent;
+  position: relative;
 }
 .language-block .rightmenu {
   top: 10px;
   margin-right: 10px;
 }
-.language-block .rightmenu select {
-  text-transform: uppercase;
+.language-block .rightmenu label {
+  color: $theme.menuLinkColor;
+  font: normal normal 400 1em/1.4em Arial,Verdana,sans-serif;
```

Comment:
Can we use `inherit` for the font family to make it easier to change fonts in the future?